### DISCUSSION TITLE
Scrollbar on Table when it gets too long?

### DISCUSSION THREAD

**Discussion by somguyth** [chakra-ui/chakra-ui#4380] [General] [answered]
I have a Grid with one of my components next to a Table; 2 columns, 1 row.

This works well when the number of entries in the table is smaller than the natural size of my other component. However, when the number of entries in Table is too long, it'll stretch my other component, which isn't nice visually. Is there a way to get a scrollbar to automatically pop up if the Table gets too long (with only the top N elements displayed by default, the rest accessible upon a scroll?)?

**Comment by anubra266** [comment]
Set a maxWidth on the table

**Reply by somguyth** [reply]
I should have spoken more clearly. I'm talking about the table height - the number of rows, not number of columns. If there's too many rows, it'll stretch my other component.

**Reply by aaronkik** [reply]
You could do something like this https://codesandbox.io/s/tender-shtern-6mmby?file=/src/index.js.

You'll need to put the table into a container and set the maxHeight on that and set overflowY to auto.

**Reply by somguyth** [reply]
> You could do something like this https://codesandbox.io/s/tender-shtern-6mmby?file=/src/index.js.
> 
> You'll need to put the table into a container and set the maxHeight on that and set overflowY to auto.

That works.

**Comment by somguyth** [comment, marked as answer]
From @AaronK97 

"
You could do something like this https://codesandbox.io/s/tender-shtern-6mmby?file=/src/index.js.

You'll need to put the table into a container and set the maxHeight on that and set overflowY to auto.
"

**Comment by maximilianotaverna** [comment]
good example , anyway of doing that without having to give the table header a background color? i.e body scroll only?
